db/config.cc

```diff
@@ -788,6 +788,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , consistent_rangemovement(this, "consistent_rangemovement", value_status::Used, true, "When set to true, range movements will be consistent. It means: 1) it will refuse to bootstrap a new node if other bootstrapping/leaving/moving nodes detected. 2) data will be streamed to a new node only from the node which is no longer responsible for the token range. Same as -Dcassandra.consistent.rangemovement in cassandra")
     , join_ring(this, "join_ring", value_status::Unused, true, "When set to true, a node will join the token ring. When set to false, a node will not join the token ring. User can use nodetool join to initiate ring joinging later. Same as -Dcassandra.join_ring in cassandra.")
     , load_ring_state(this, "load_ring_state", value_status::Used, true, "When set to true, load tokens and host_ids previously saved. Same as -Dcassandra.load_ring_state in cassandra.")
+    , replace_node_first_boot(this, "replace_node_first_boot", value_status::Used, "", "The Host ID of the dead node to replace. If this node has been bootstrapped successfully, this option will be ignored.")
```

Comment:
What is this node? Is "this node" the node to replace or the node that has been started with this option? Would be really nice to disambiguate.

Reply:
It's the replacing node. How about the following:

The Host ID of a dead node to replace. If the replacing node has already been bootstrapped successfully, this option will be ignored.

Reply:
Rephrased in [07d8629](https://github.com/scylladb/scylladb/commit/07d8629e2bd922483cb2000d26f5bf285def0b9a)

Reply:
It still says "this node".

Reply:
oops, will change.